### PR TITLE
Add default path for 'BATS_LIB_PATH' environment variable'

### PR DESCRIPTION
**Pull request by hwittenborn**
- [x] I have reviewed the [Contributor Guidelines][contributor].
- [x] I have reviewed the [Code of Conduct][coc] and agree to abide by it

[contributor]: https://github.com/bats-core/bats-core/blob/master/docs/CONTRIBUTING.md
[coc]:         https://github.com/bats-core/bats-core/blob/master/docs/CODE_OF_CONDUCT.md

### CODE REVIEW

**Review by martin-schulze-vireso — APPROVED** [review]

**Comment by martin-schulze-vireso** [comment]
Thanks for your contribution.

**Comment by hwittenborn** [comment]
@martin-schulze-vireso: Any idea when the next release will be made? I'd like to start using this in my tests but I need to deploy it via my Debian package first.

**Comment by martin-schulze-vireso** [comment]
Well, the deadline for 1.8.0 is this month. However, I want to close all issues in the milestone.

File: libexec/bats-core/bats [hunk 1]
```diff
@@ -101,6 +101,7 @@ export BATS_TMPDIR="${TMPDIR:-/tmp}"
 BATS_TMPDIR=${BATS_TMPDIR%/} # chop off trailing / to avoid duplication
 export BATS_RUN_TMPDIR=
 export BATS_GUARANTEED_MINIMUM_VERSION=0.0.0
+export BATS_LIB_PATH=${BATS_LIB_PATH-/usr/lib/bats}
```
Comment (jcassee) [inline libexec/bats-core/bats:104]: Sorry for the drive-by comment, but shouldn't this be `${BATS_LIB_PATH:-/usr/lib/bats}`? (Note the colon.)

Comment (martin-schulze-vireso) [reply libexec/bats-core/bats:104]: I only translated this from the original Code but I assumed the intention is to allow for users to specify an empty search path. This would mit ve possible with colon.

Comment (jcassee) [reply libexec/bats-core/bats:104]: Sure, but the syntax for that is `${var:-default}`, not `${var-default}`, right? ([Bash docs](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html))

Comment (martin-schulze-vireso) [reply libexec/bats-core/bats:104]: No, :- is alternative value for unset or empty, - only replaces unset variables.

Comment (jcassee) [reply libexec/bats-core/bats:104]: I did not know that. Thank you for the explanation!
